modules/rotating-certificate-authority.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rotating-certificate-authority_{context}"]
+== Rotating the etcd certificate
+
+Rotate the `etcd` certificate before it expires.
+
+.Procedure
+
+. Verify the remaining lifetime of the new signer certificate by running the following command:
++
+[source,terminal]
+----
+$ oc get secret -n openshift-etcd etcd-signer -ojsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+----
+
+. Re-create the signer, if the remaining lifetime is close to the current date, by deleting the signer and wait for the static pod rollout by running the following commands:
```

Comment:
I think that putting the lifetime phrase first ties in better with the previous step. 
```suggestion
. If the remaining lifetime is close to the current date, re-create the signer by deleting the signer and wait for the static pod rollout by running the following commands:
```
What should the user do if the remaining lifetime is **_not_** close to the current date?